### ISSUE TITLE
Remote Content Browsing: Replace indeterminate downloading loader by determinate on a content page

### ISSUE THREAD

**Issue by MisRob** [learningequality/kolibri#10565]
A follow-up to https://github.com/learningequality/kolibri/pull/10558

## Blocked by

#10391
#10564 

## Summary

When downloading remote content on a content page, we currently display an indeterminate loader (related PR https://github.com/learningequality/kolibri/pull/10558):
![download](https://user-images.githubusercontent.com/13509191/234488155-1488598d-d01d-4273-b357-329a46189e6c.gif)

This should be replaced by a determinate loader ([decision reference](https://github.com/learningequality/kolibri/pull/10285#issuecomment-1486716508)) as soon as backend can provide us with granular progress data (#10391)

Reported as a separate issue from #10564 since we agreed for the determinate loader version to be a stretch goal.
